lib/service.js

```diff
@@ -1702,24 +1702,43 @@
          * `splunkjs.Service.Job` instances.
          *
          * @example
-         *
+         *      With callback 
+         * 
```

Comment:
Hey there, could you actually remove the changes made in https://github.com/splunk/splunk-sdk-javascript/pull/95? I'm trying to merge that PR into your branch so that the contributor can get credit for the original changes and just keep the testing changes that you made. Let me know if you have any questions or concerns

Reply:
sure thing.

Reply:
perfect, thanks!